docs/src/routes/docs/components/checkbox-chip.svx

```diff
@@ -0,0 +1,107 @@
+---
+name: CheckboxChip
+relatedComponents:
+  - name: Chip
+    link: ./docs/components/chip
+  - name: RadioChip
+    link: ./docs/components/radio-chip
+  - name: RadioChipGroup
+    link: ./docs/components/radio-chip-group
+  - name: CheckboxChipGroup
+    link: ./docs/components/checkbox-chip-group
+---
+
+<script>
+  import { CheckboxChip } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+</script>
+
+# CheckboxChip
+
+An option to select in an inline fashion, as opposed to regular checkboxes.
+
+If you don't need too much customization, consider using [==`CheckboxChipGroup`==](./docs/components/checkbox-chip-group), which offers a nice way to generate lists of options.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <CheckboxChip value="wrong" name="selection" outline>opt 1</CheckboxChip>
+    <CheckboxChip value="right" name="selection" small>opt 2</CheckboxChip>
+    <CheckboxChip value="opt 3" name="selection" />
+  </div>
+  <div slot="source">
+
+```svelte
+<CheckboxChip value="wrong" name="selection" outline>opt 1</CheckboxChip>
+<CheckboxChip value="right" name="selection" small>opt 2</CheckboxChip>
+<CheckboxChip value="opt 3" name="selection" />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`small`** | `false` | `boolean` | Makes the chip smaller. |
+| **`outline`** | `false` | `boolean` | Adds a border to the chip instead of the background. |
+| **`noPadding`** | `false` | `boolean` | Clears the internal padding. Useful for adding deletion buttons. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`value`**==&nbsp;<span class="required">*</span> |  | `any` | The value of this checkbox. Included in events. |
+| ==**`checked`**== | `false` | `boolean` | Whether the chip is selected or not. |
+| ==**`name`**==&nbsp;<span class="required">*</span> |  | `string` | The name of the group to relate this checkbox chip to. |
+| **`disabled`** | `false` | `boolean` | Disables the chip and disallows selection. |
+| **`title`** | `null` | `string` | The tooltip to give to a chip. |
+| **`...`** |  | `any` | Everything else will be passed to the underlying `<input>` element. |
+
+<div class="required">* Required</div>
+
+### Class Properties
+
+All `<CheckboxChip>` components use the `.input-chip.checkbox-chip` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. Falsy values disregarded. |
+| **`inputClass`** | `null` | `string` | A class string to add to the `<input>` element. |
```

Comment:
Why does this one not say "Falsey values disregarded." like the rest? If it's because it doesn't disregard them, how about making it consistent with them? And then perhaps move this comment outside the table since it is common to all entries

Reply:
Yeah, that's currently a problem. Not all of them disregard falsy values, which means we must be explicit about which ones do. I do intend to bring consistency to this though, and then we'll remove it.